tests/unit/App.spec.js

```diff
@@ -269,4 +269,38 @@ Given("an App", () => {
       });
     });
   });
+
+  When("a response with link headers is returned", () => {
+    const rawLinks =
+      "</>; rel=\"http://example.com\"; title=\"Example link\", " +
+      "</test>; rel=\"http://test.com\"; title=\"Test link\"";
+
+    beforeEach(() => {
+      app.setMethods({
+        request: app.vm.setLinks
+      });
+
+      app.vm.request(rawLinks);
+      app.update();
+    });
+
+    Then("one link should be displayed", () => {
+      expect(app.text().includes("Example link")).to.be.true;
+    });
+
+    Then("all the links should be displayed", () => {
+      expect(app.text().includes("Test link")).to.be.true;
```

Comment:
This doesn't seem to test what it says it does.

Reply:
@jdesrosiers, I tried to find some way to count all anchor tags in a component, but couldn't figure out how to do so. Could I just change the test name for now, to `"the second link should be displayed"`?

Reply:
Yes, that's fine.